tests/read_group/test_read_group_resources.py

```diff
@@ -196,92 +135,3 @@ def _create_save_to_db(self):
         db.session.commit()
 
         return rg
-
-    def test_filter_by_gf(self):
```

Comment:
why delete these tests?

Reply:
These tests are no longer needed since we've removed the functionality that it tests. You can no longer go to the /genomic-files endpoint and filter by read_group_id nor can you go to the /read-groups endpoint and filter by by genomic_file_id. You'll now have to go to the /read-group-genomic-files endpoint to filter associations

Reply:
The general filtering tests which include  /read-group-genomic-files are in tests/test_filter_params.py

Reply:
lol ok